src/Validation/Validator.php

```diff
@@ -195,6 +200,16 @@ public function provider($name, $object = null)
     }
 
     /**
+     * Get the list of providers in this validator.
+     *
+     * @return array
+     */
+    public function providers()
+    {
+        return array_keys($this->_providers);
```

Comment:
I figured we could return the names and let consumers get the providers with provider().